### PR TITLE
Update jtreg to 7.1.1+1 for JDK 20

### PR DESCRIPTION
**Pull request by luhenry**

### CODE REVIEW

**Comment by smlambert** [comment]
Links to jdk20 builds at https://trss.adoptium.net/builds?buildName=openjdk20-pipeline&url=https%3A%2F%2Fci.adoptopenjdk.net%2Fjob%2Fbuild-scripts&type=Test 

(and riscv builds at: https://trss.adoptium.net/builds?buildName=jdk-linux-riscv64-temurin&url=https%3A%2F%2Fci.adoptopenjdk.net%2Fjob%2Fbuild-scripts%2Fjob%2Fjobs%2Fjob%2Fjdk&type=Test)

**Comment by llxia** [comment]
`test.getDependency` job failed with PR https://github.com/adoptium/TKG/pull/381. @luhenry could you take a look? Thanks

https://ci.adoptopenjdk.net/job/test.getDependency/1649/console

```
--> file downloaded to ./jtreg_7_1_1_1.tar.gz
downloading https://ci.adoptopenjdk.net/job/dependency_pipeline/lastSuccessfulBuild/artifact/jtreg/jtreg-7-1-1+1.tar.gz.sha256sum.txt
ERROR: sha checksum error.
--> file downloaded to ./jtreg_7_1_1_1.tar.gz.sha256sum.txt
Expected sha is: <html>
<head>
<meta,
Actual sha is  : cfb3763e85bc633ca37d72f7843fb19e29ebda1bb189b2fc9244eaaadd2fc53d.
Please delete ./jtreg_7_1_1_1.tar.gz and rerun the program!

```

**Comment by llxia** [comment]
I do not see https://ci.adoptopenjdk.net/job/dependency_pipeline/lastSuccessfulBuild/artifact/jtreg/jtreg-7-1-1+1.tar.gz or https://ci.adoptopenjdk.net/job/dependency_pipeline/lastSuccessfulBuild/artifact/jtreg/jtreg-7-1-1+1.tar.gz.sha256sum.txt

**Comment by luhenry** [comment]
The jenkins URL was wrong, the correct one is done with https://github.com/adoptium/TKG/pull/382

**Comment by llxia** [comment]
Thanks @luhenry 
Note: this PR can only be tested and merged after https://github.com/adoptium/TKG/pull/382 is merged.

File: openjdk/build.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@
 							<property name="jtregTar" value="jtreg_6_1_1"/>
 						</then>
 						<else>
-							<property name="jtregTar" value="jtreg_7_1"/>
+							<property name="jtregTar" value="jtreg_7_1_1_1"/>
```
Comment (sophia-guo) [inline openjdk/build.xml:50]: Does this mean jtreg_7_1 can be replaced by jtreg_7_1_1_1? So we can do a cleanup with jtreg_7_1? @luhenry

**Comment by luhenry** [comment]
https://ci.adoptopenjdk.net/job/test.getDependency/1651/ has run successfully

File: openjdk/build.xml (hunk 1 above)
Comment (luhenry) [reply openjdk/build.xml:50]: Once this PR has been merged I expect we can remove `jtreg_7_1` at https://github.com/adoptium/TKG/pull/381/files#diff-6548ada249042f87589ac5683c2f76bd9b8447695fd19fd02ad3030a2592b775R146-R152 and https://github.com/adoptium/ci-jenkins-pipelines/pull/532/files#diff-a0facb89f6e82d0bdeb77e87b5306d0a9d185eef333ba542dd83cee60afb885cR128

**Comment by llxia** [comment]
JDK20 jdk_math: https://ci.adoptopenjdk.net/job/Grinder/6275

**Review by llxia — APPROVED** [review]
LGTM

**Pull request merged**